rasa/shared/core/domain.py

```diff
@@ -628,24 +629,12 @@ def _initialize_forms(
             `FormAction` which is implemented in the Rasa SDK.
         """
         if isinstance(forms, dict):
+            for form_name, form_data in forms.items():
```

Comment:
do we have a test for this behavior?

Reply:
I can't find the tests for it - how is it called?

Reply:
@Imod7

Reply:
> What behavior do we want to test here?

We should test that the Domain is read correctly if users have a (deprecated) domain like this:

```yaml

forms:
  my_form:
    my_slot:
    - type: from_text
```

Reply:
This behaviour is checked with this test `test_form_with_no_required_slots_keyword`.